index.css

```diff
@@ -0,0 +1,144 @@
+body {
+  align-items: center;
+  background-color: #343644;
+  display: flex;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  height: 100vh;
+  justify-content: center;
+}
+
+.main {
```

Comment:
So Gabriel, in class we have been told to use the lowest specificity needed for every single selector and we only use on main element per website (most of the times) so we can actually just use main selector element instead.

Reply:
Foo